web/src/components/tasks/business-formation/BusinessFormation.test.tsx

```diff
@@ -296,7 +296,7 @@ describe("<BusinessFormation />", () => {
     page.fillText("Address line1", "1234 main street");
     page.fillText("Address line2", "Suite 304");
     page.fillText("Address zip code", "08001");
-    page.selectByText("Address municipality", "Newark");
+    page.fillText("Address municipality", "Newark");
```

Comment:
boulder - Why is this method being changed? Isn't it a part of th main business address?